src/pages/strategy/edit/overlapping/index.tsx

```diff
@@ -0,0 +1,3 @@
+export const EditOverlappingStrategyPage = () => {
+  return <></>;
```

Comment:
what is this needed for?

Reply:
An artifact from an old time when a dev thought he could use this PR as a base for a better place where each strategy type would have its own page. The delusional dev soon realized that it was a chimera and, in front of the complexity lying before him, decided to focus his energy into the sole task of editing overlapping strategies. In his journey he forgot the ground is initially prepared.

Hopefully you were here to ask him to clean his shit XD